_protected/app/system/modules/mail/views/base/tpl/list.inc.tpl

```diff
@@ -22,16 +33,21 @@
                 {{ $move_to = ($is_delete) ? 'delete' : 'trash' }}
                 {{ $label_txt = ($is_delete) ? t('Delete') : t('Trash') }}
 
+                {{ $subject = escape(str_replace('re ', '', $msg->title), true) }}
```

Comment:
For your information, here you just replace the "re" lower case. It's case-sentitive. You will have to use `str_ireplace()` for case-insensitive replace https://www.php.net/manual/en/function.str-ireplace.php

Reply:
Oh okay wow I didn't know that. I thought str_replace was for complex situations where you may need to use regex and preg was for more simple use.  I like it when I learn something like this 👍

Reply:
👍 
No, actually, it's the other way around. `preg_replace` with regex would be for more complex scenarios. When it's possible, it's always better to use str_(i)replace since it's more readable (regex aren't easy to read) and it's faster too.